### PR TITLE
Release notes for Release 3.5

### PR DESCRIPTION
**Pull request by awholegunch**
Running as [Quick Build 3](http://builds.cask.co/browse/CDAP-DQB89-3)

[Release Notes](http://builds.cask.co/artifact/CDAP-DQB89/shared/build-3/Docs-HTML/3.5.0-SNAPSHOT/en/reference-manual/release-notes.html#release-3-5-0)

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -23,6 +23,574 @@ Cask Data Application Platform Release Notes
...
+- :cask-issue:`CDAP-6929` - Logs from Spark driver and executors are now collected.
+
+- :cask-issue:`CDAP-6930` - **Incompatible Change:** Updated the "cdap-clients" to throw
+  ``UnauthorizedException`` when an operation returns ``403 - Forbidden`` from CDAP. Users
+  of "cdap-clients" may need to update their code to handle these exceptions.
+
+- :cask-issue:`CDAP-6935` - Fix a bug where the live-info endpoint was not working for
+  Workflows, MapReduce, Worker, and Spark.
+
+- :cask-issue:`CDAP-6939` - Fixed support in the CDAP UI for Google Chrome prior to
```
Comment (nitinmotgi) [inline cdap-docs/reference-manual/source/release-notes.rst]: Fixed support -- should this be -- Added CDAP UI support for Google Chrome releases prior to version 44.

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -23,6 +23,574 @@ Cask Data Application Platform Release Notes
...
+  true. TTL for cache entries (``security.authorization.cache.ttl.secs``) as well as refresh
+  interval (``security.authorization.cache.refresh.interval.secs``) is also configurable.
+
+- :cask-issue:`CDAP-5740` - Provided access to ``Partitioner`` and ``Comparator`` classes
+  to the ``MapReduceTaskContext`` by implementing ``ProgramLifeCycle``.
+
+- :cask-issue:`CDAP-5770` - Allow setting of YARN container resources requirements for all
+  program types via preferences and runtime arguments.
+
+- :cask-issue:`CDAP-6052` - Fixed a bug that prevented the setting of ``local.data.dir``
```
Comment (nitinmotgi) [inline cdap-docs/reference-manual/source/release-notes.rst]: This is not a new feature right ?

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/release-notes.rst]: Fixed.

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/release-notes.rst]: No. There are a bunch of these (notes in the wrong category). Fixed and fixing the others.

**Pull request merged**